packages/lab/src/list-next/useList.ts

```diff
@@ -172,16 +197,22 @@ export const useList = ({
   // HANDLERS
   const blurHandler = () => {
     setShowFocusRing(false);
+    setFromMouse(false);
   };
 
   const mouseDownHandler = () => {
     // When list gets focused, we can't guarantee that focus happens after click event.
-    // If first focus (where !activeDescendant) happens from a click, list shouldn't render focus ring in the first element.
-    setShowFocusRing(false);
+    // If focus happens from a click, list shouldn't render focus ring.
```

Comment:
on controlled story I get a focus ring on mouse click. is that intended on controlled vs un?

Reply:
i need to double check this with Design